### PR TITLE
[nnfwapi] Test : Add Tile one-op test

### PR DESCRIPTION
**Pull request by wateret**
Add Tile one-op test

- 2 positive tests (1 disabled)
- 1 negative test

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

File: tests/nnfw_api/src/CircleGen.cc [hunk 1]
```diff
@@ -205,6 +205,13 @@ uint32_t CircleGen::addOperatorSplit(const OperatorParams &params, int32_t num_s
                                 circle::BuiltinOptions_SplitOptions, options);
 }
 
+uint32_t CircleGen::addOperatorTile(const OperatorParams &params)
+{
+  auto options = circle::CreateRankOptions(_fbb).Union();
```
Comment (hseok-oh) [inline tests/nnfw_api/src/CircleGen.cc:210]: `CreateRankOptions` -> `CreateTileOptions`

Comment (wateret) [reply tests/nnfw_api/src/CircleGen.cc:210]: Thanks! but it should be removed, as there is no `CreateTileOptions`.

Comment (hseok-oh) [reply tests/nnfw_api/src/CircleGen.cc:210]: `circle::CreateTileOptions` :
`runtime/onert/frontend/circle_schema/include/circle_schema_generated.h:5528`

Comment (wateret) [reply tests/nnfw_api/src/CircleGen.cc:210]: Oops, I might looked at an old schema 😓

**Comment by wateret** [comment]
@hseok-oh Updated, PTAL

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**